content/src/storage/RepositoryQueue.ts

```diff
@@ -7,22 +26,42 @@ import PQueue from 'p-queue'
 export class RepositoryQueue {
   public static readonly TOO_MANY_QUEUED_ERROR =
     'There are too many requests being made to the database right now. Please try again in a few minutes'
+  public static readonly DEFAULT_MAX_CONCURRENCY = 20
+  public static readonly DEFAULT_MAX_QUEUED = 100
```

Comment:
We are now setting this to 100 (instead of 50 like previously) so we can study how it works in production, and choose a number then